pymc/tests/test_sampling.py

```diff
@@ -1101,6 +1101,7 @@ def test_logging_sampled_basic_rvs_posterior_mutable(self, caplog):
         # hasn't changed since sampling
         idata = az_from_dict(posterior={"y1": np.zeros(5), "y2": np.zeros(5)})
         with m:
+            m.set_data("x1", 1)
```

Comment:
The condition should be changed to not expect resampling (we have one for resampling immediately below), and the comments above updated (they are outdated now as well). 

Or perhaps just remove this test in favor of the new one you added?

Reply:
I'll remove this test in favor of the new one